fctesting/sandbox/Dockerfile

```diff
@@ -0,0 +1,44 @@
+ARG TMP_BUILD_DIR=/tmp/build
+
+FROM rust:1.32 as build
+
+ARG TMP_BUILD_DIR
```

Comment:
Did you mean to declare this `ARG` a second time?

Reply:
Yea, if I don't say that variable is in this scope it doesn't exist. I hope that makes sense.

Reply:
From the [documentation](https://docs.docker.com/engine/reference/builder/#arg):

> The `ARG` instruction defines a variable that users can pass at build-time to the builder with the `docker build` command using the `--build-arg <varname>=<value>` flag. If a user specifies a build argument that was not defined in the Dockerfile, the build outputs a warning.

If you're not using the `ARG` command to allow overrides with `--build-arg`, please replace it with `ENV`.